docs/releases.rst

```diff
@@ -0,0 +1,109 @@
+.. _releases:
+
+Releases
+=========
+
+There are three types of Renku releases: Sprint release, Hot/bug fix release and Quick release.
+
+Sprint release
+---------------
+
+A new release is planned on the Tuesday after the end of every sprint at 12:00
+(every three weeks).
+The current state of the main branch is released as is, only checking the
+wording in the CHANGELOG to ensure it is of good quality and adding the version
+information.
+
+Pre-requisites [Graph, UI, RP]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Components release whenever they want, the schedule is up to them. They are
+responsible for updating the changelog in the auto-PR in the renku repository,
+in a section above the one for the most recent release.
+The auto-PR is only merged with an updated changelog/release notes.
+Ideally, components would release when there are meaningful changes or so

```

Comment:
This sentence is confusing. I think you can just leave it at "the schedule is up to them"